lib/fluent/plugin/parser_json.rb

```diff
@@ -30,6 +30,11 @@ class JSONParser < Parser
       desc 'Set JSON parser'
       config_param :json_parser, :enum, list: [:oj, :yajl, :json], default: :oj
 
+      # The Yajl library defines a default buffer size of 8092 when parsing
+      # from IO streams, so maintain this for backwards-compatibility.
+      # https://www.rubydoc.info/github/brianmario/yajl-ruby/Yajl%2FParser:parse
+      config_param :stream_buffer_size, :integer, default: 8092
```

Comment:
Please add `:desc` parameter to explain the parameter.  This is only for `yajl`.

Reply:
Done!

Reply:
If you use `desc` method, not `:desc` parameter, it should be put on top of `config_param`.
Currently, `desc 'Set the buffer size that Yajl will use when parsing streaming input'` is for `time_tupe`, not `stream_buffer_size`.
See https://docs.fluentd.org/v1.0/articles/api-plugin-base

Reply:
Ah, that was a silly mistake, thanks for the pointer! Fixed.